dts/arm/st/u0/stm32u0.dtsi

```diff
@@ -387,6 +387,16 @@
 				status = "disabled";
 			};
 		};
+
+		tsc: tsc@40024000 {
+			compatible = "st,stm32-tsc";
+			reg = <0x40024000 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x01000000>;
```

Comment:
Please use new macro: 
```suggestion
			clocks = <&rcc STM32_CLOCK(AHB1, 24U)>;
```

Reply:
This is not yet in my branch, will rebase